Add Wilfred profile

Wilfred007 opened the pull request:
## Description

I've made all the changes and updates

- Added links to my socials
- Spaced the socials icons
- Replaced the screenshot
- Changed the image name to my EVM address
- Moved static data outside React component
- I fixed nextjs image error
- I moved the ADDRESS below the import statements
- I've fixed the missing props error

![Screenshot_16-1-2025_65136_super-duper-giggle-pjjjg79rjpwc7jjp-3000 app github dev](https://github.com/user-attachments/assets/f7bf1205-9c98-49d8-9eb7-f8761cf6ac6e)

## Additional Information

- [x] I have read the [contributing docs](/scaffold-eth/scaffold-eth-2/blob/main/CONTRIBUTING.md) (if this is your first contribution)
- [x] This is not a duplicate of any [existing pull request](https://github.com/scaffold-eth/scaffold-eth-2/pulls)


Your ENS/address: 0x2aA2f7090f0ADD72B2d50386CdDE97CE27898287

derrekcoleman commented on packages/nextjs/app/builders/0x2aA2f7090f0ADD72B2d50386CdDE97CE27898287/page.tsx:
```diff
@@ -0,0 +1,92 @@
+                fascinated by the potential of blockchain technology to reshape our digital landscape.
+              </p>
+              <p>
+                Currently, I&apos;m working on projects that combines NFTs markets and upcomiing artist, allowing users
```
typo: upcomiing

derrekcoleman reviewed (changes requested):
Thank you for your PR, Wildfred007! And welcome to the Batch12 repo :)

Thank you for including a screenshot of your PR 📷 
- Please also add a few summary sentences to your PR message describing what this PR adds (your page, an overview of sections, keep it high-level).
- Also read the [Contributing](https://github.com/scaffold-eth/scaffold-eth-2/blob/main/CONTRIBUTING.md) docs, then check both boxes in the PR message to acknowledge them.

I look forward to buidling with you!

Wilfred007 commented:
I've made all the changes and updates

- Added links to my socials
- Spaced the social icons
- Replaced the screenshot
- Changed the image name to my EVM address

@derrekcoleman 

![Screenshot_16-1-2025_65136_super-duper-giggle-pjjjg79rjpwc7jjp-3000 app github dev](https://github.com/user-attachments/assets/f7bf1205-9c98-49d8-9eb7-f8761cf6ac6e)

derrekcoleman commented on packages/nextjs/app/builders/0x2aA2f7090f0ADD72B2d50386CdDE97CE27898287/page.tsx:
```diff
@@ -0,0 +1,96 @@
+import { Address } from "~~/components/scaffold-eth";
+
+const WilfredProfile: NextPage = () => {
+  const address = "0x2aA2f7090f0ADD72B2d50386CdDE97CE27898287";
+
+  // Links data array
+  const linksData = [
+    {
+      href: "https://github.com/wilfred007",
+      label: "GitHub",
+      icon: <FaGithub className="mt-1 mx-2" />,
+    },
+    {
+      href: "https://twitter.com/drips_young",
+      label: "Twitter",
+      icon: <FaXTwitter className="mt-1 mx-2" />,
+    },
+    {
+      href: "https://www.linkedin.com/in/wilfred-adzer-89261923a/",
+      label: "LinkedIn",
+      icon: <FaLinkedin className="mt-1 mx-2" />,
+    },
+  ];
+
+  // Reusable LinkButton component
+  const LinkButton = ({ href, label, icon }: { href: string; label: string; icon: React.ReactNode }) => (
+    <Link
+      href={href}
+      target="_blank"
+      rel="noopener noreferrer"
+      className="bg-blue-500 hover:bg-blue-600 text-white flex font-bold py-2 px-4 rounded"
+    >
+      {icon}
+      {label}
+    </Link>
+  );
```
Please move all of your static data and reusable UI components outside of the `NextPage` component. Cut line 8 and paste it on line 42, above the `return` statement.

derrekcoleman reviewed (changes requested):
One last change, following up on [this request](https://github.com/BuidlGuidl/batch12.buidlguidl.com/pull/34/files#r1917254965) from last week. lmk if you aren't sure what I mean

phipsae commented on packages/nextjs/app/builders/0x2aA2f7090f0ADD72B2d50386CdDE97CE27898287/page.tsx:
```diff
@@ -0,0 +1,92 @@
+import { FaXTwitter } from "react-icons/fa6";
+
+const BuildersProfile = () => {
+  const address = "0x2aA2f7090f0ADD72B2d50386CdDE97CE27898287";
```
Still outstanding

phipsae commented on packages/nextjs/app/builders/0x2aA2f7090f0ADD72B2d50386CdDE97CE27898287/page.tsx:
```diff
@@ -0,0 +1,94 @@
+];
+
+const WilfredProfile: NextPage = () => {
+  const LinkButton = ({ href, label, icon }: { href: string; label: string; icon: React.ReactNode }) => (
+    <Link
+      href={href}
+      target="_blank"
+      rel="noopener noreferrer"
+      className="bg-blue-500 hover:bg-blue-600 text-white flex font-bold py-2 px-4 rounded"
+    >
+      {icon}
+      {label}
+    </Link>
+  );
+
+  const ADDRESS = "0x2aA2f7090f0ADD72B2d50386CdDE97CE27898287";
```
The component and the static constant you could place outside your profile page component (like @derrekcoleman suggested already).

phipsae commented on packages/nextjs/app/builders/0x2aA2f7090f0ADD72B2d50386CdDE97CE27898287/page.tsx:
```diff
@@ -0,0 +1,94 @@
+      <div className="flex flex-col items-center gap-8 py-12 px-4 max-w-4xl mx-auto">
+        <div className="flex flex-col items-center gap-4">
+          <div className="relative w-32 h-32">
+            <Image
+              src="/0x2aA2f7090f0ADD72B2d50386CdDE97CE27898287.jpg"
+              alt="0x2aA2f7090f0ADD72B2d50386CdDE97CE27898287"
+              layout="fill"
+              className="rounded-full ring-4 ring-blue-500"
+            />
```
As a good practice it is recommended to check to console for any warnings/errors.
It shows me the following:
<img width="413" alt="Screenshot 2025-01-28 at 09 43 17" src="https://github.com/user-attachments/assets/dc44510f-36e6-4368-b061-7a89fbe09b57" />

Pls make sure to fix them.

phipsae replied:
This is still outstanding

phipsae replied:
<img width="418" alt="Screenshot 2025-01-29 at 20 14 05" src="https://github.com/user-attachments/assets/ef0e6717-1e31-45f3-a8cc-93bab88ce48f" />

phipsae reviewed (changes requested):
Sorry for the wait.

Below you'll find two tiny comments and then we finally merge.

Next time you implement your changes, you can request a review from us directly on GitHub. Start by leaving a quick comment to let us know you've completed the changes. Then, in the top-right corner of the page, locate the reviewer box and click on the 🔄 icon to request a review.

Thank you!

derrekcoleman commented:
Thanks for addressing the Image prop issue, @Wilfred007! Please take a look at the other comment, related to factoring *all* of your constants outside of the NextPage component - you still have an ADDRESS in the middle that would be more organized if it was just below your import statements.

derrekcoleman reviewed (changes requested):
Thanks for addressing the Image prop issue, @Wilfred007! Please take a look at the other comment, related to factoring all of your constants outside of the NextPage component - you still have an ADDRESS in the middle that would be more organized if it was just below your import statements.

Wilfred007 commented:
> Thanks for addressing the Image prop issue, @Wilfred007! Please take a look at the other comment, related to factoring all of your constants outside of the NextPage component - you still have an ADDRESS in the middle that would be more organized if it was just below your import statements.

Thank you for your patience, I've mad the changes

phipsae commented:
@Wilfred007, please see my comments below. I’m still seeing the warning in the console. In general, when requesting a review, please ensure that all previous points have been addressed and check the console for any warnings/erros. Otherwise, it becomes repetitive to highlight the same issues multiple times.

phipsae commented:
@Wilfred007 why do you request a review, when you haven't made any chances? I don't see any new commits...

This is the error it get still shown to me. If you scroll up, I already mentioned it in the unresolved conversation. 
![image](https://github.com/user-attachments/assets/9fe2aa50-ec8f-474d-9d29-a79004621a48)

Pls fix it, thank!

derrekcoleman commented:
> Thank you for your patience, Ive fixed the missing props error, but I'm seeing a lit is in dev mode warning is that something I need to fix too because I've not seen this warning before. Can you help? Thank you

That error does not seem to happen in [production](https://batch12buidlguidl-10k0w0fso-buidlguidldao.vercel.app/), which is consistent with it being a "dev mode" (i.e. local testing) warning. Have you searched the error on the internet or asked an AI what it might mean and whether any changes might be needed? Those are steps I sometimes take when I'm facing a new, unfamiliar error.

Wilfred007 commented:
> > Thank you for your patience, Ive fixed the missing props error, but I'm seeing a lit is in dev mode warning is that something I need to fix too because I've not seen this warning before. Can you help? Thank you
> 
> That error does not seem to happen in [production](https://batch12buidlguidl-10k0w0fso-buidlguidldao.vercel.app/), which is consistent with it being a "dev mode" (i.e. local testing) warning. Have you searched the error on the internet or asked an AI what it might mean and whether any changes might be needed? Those are steps I sometimes take when I'm facing a new, unfamiliar error.

I did but can't seem to find a direct solution, it's saying it might not be something serious in production. Is it possible for us to move on with this?

derrekcoleman reviewed (approved):
Thanks for taking the time to implement our suggestions, @Wilfred007! Your git-fu skills are stronger for it 🥷
